src/Integration/ProjectPropertyManager.cs

```diff
@@ -0,0 +1,123 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ProjectPropertyManager.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using EnvDTE;
+using System;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    [Export(typeof(IProjectPropertyManager))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    internal class ProjectPropertyManager : IProjectPropertyManager
```

Comment:
No, because the only way to get access to local services is through the `IHost`, which is internal.
Making the `IHost` public leads to a whole host of other access modifier changes.